.simplecov

```diff
@@ -0,0 +1,40 @@
+# RM_INFO is set when using Rubymine.  In Rubymine, starting SimpleCov is
+# controlled by running with coverage, so don't explicitly start coverage (and
+# therefore generate a report) when in Rubymine.  This _will_ generate a report
+# whenever `rake spec` is run.
+SimpleCov.start unless ENV['RM_INFO']
```

Comment:
I'm still not happy introducing code which is only for a "single" developer's development environment - no offense :smile:.

```ruby
SimpleCov.start unless ENV.key? 'ARUBA_NO_COVERAGE'
```
would be ok for me. You can then setup a rake task which sets `ENV['ARUBA_NO_COVERAGE']` if `ENV['RM_INFO']` is set.

We would set the `ARUBA_COVERAGE` then by default in our rake test task.


@mattwynne @jarl-dk 

What do you think?